### ISSUE TITLE
使用最新版belle对baichuan2-7b进行sft，lr为0

### ISSUE THREAD

**Issue by LucasJau** [baichuan-inc/Baichuan2#161]
使用最新版belle对baichuan2-7b进行sft全程lr=0，训练步数超过warmup步数之后依旧提示：
tried to get lr value before scheduler/optimizer started stepping, returning lr=0

运行脚本：
![image](https://github.com/baichuan-inc/Baichuan2/assets/13137377/3dca800e-0826-4d22-a05c-36372edbe656)

ds config：
![image](https://github.com/baichuan-inc/Baichuan2/assets/13137377/735ed83a-3210-4f1b-a2ac-e6c037125d00)

GPU：8 * 40GA100

**Comment by mmmans** [comment]
What's your problem?

**Comment by LucasJau** [comment]
> What's your problem?

Learning rate keeps equaling 0 during the training.
Like this (the snapshot isn't mine, but the warnings are the same)
[WARNING|trainer_pt_utils.py:803] 2021-11-26 13:28:52,075 >> tried to get lr value before scheduler/optimizer started stepping, returning lr=0
[WARNING|trainer_pt_utils.py:803] 2021-11-26 13:28:52,076 >> tried to get lr value before scheduler/optimizer started stepping, returning lr=0
[WARNING|trainer_pt_utils.py:803] 2021-11-26 13:28:52,076 >> tried to get lr value before scheduler/optimizer started stepping, returning lr=0
{'loss': 0.0729, 'learning_rate': 0, 'epoch': 0.02}                                                                                                                                                         
  4%|██████▋                                                                                                                                                         | 1000/24128 [06:12<2:20:51,  2.74it/s][WARNING|trainer_pt_utils.py:803] 2021-11-26 13:31:56,044 >> tried to get lr value before scheduler/optimizer started stepping, returning lr=0
[WARNING|trainer_pt_utils.py:803] 2021-11-26 13:31:56,044 >> tried to get lr value before scheduler/optimizer started stepping, returning lr=0
[WARNING|trainer_pt_utils.py:803] 2021-11-26 13:31:56,045 >> tried to get lr value before scheduler/optimizer started stepping, returning lr=0
[WARNING|trainer_pt_utils.py:803] 2021-11-26 13:31:56,045 >> tried to get lr value before scheduler/optimizer started stepping, returning lr=0
{'loss': 0.0765, 'learning_rate': 0, 'epoch': 0.04}

**Comment by mmmans** [comment]
refer to https://github.com/LianjiaTech/BELLE/issues/134

**Comment by LucasJau** [comment]
> refer to [LianjiaTech/BELLE#134](https://github.com/LianjiaTech/BELLE/issues/134)

It doesn't fit my issue. The newest BELLE need transformer>=4.32.1 and deepspeed>=0.9.0.

**Comment by frankang** [comment]
这是loss崩了的体现（可能是belle加载pretrained model哪里有bug），换baichuan2自己的finetune代码就没问题了。
